lifter-data/name-corrections.dat

```diff
@@ -78,12 +83,12 @@ Alexandra De L'Armour,Alexandra DeL'Armour,Alexandra De L'armour
 Alexandra Murphy-Jelley,Alexandra Murphy Jelley
 Alexandre Pelletier,Alexandre Pellletier,Alexand Pelletier
 Alexis Guzmán,Alexis Guzman
-Alf-Ivar Hellerud,Alf Ivar Hellerud
```

Comment:
The changes in this file remove a large number of existing name corrections. Was that intentional? If yes, why?

Reply:
Not intentional, due to this pull being based on an out of date upstream. Will integrate the existing corrections.